git/attribs.go

```diff
@@ -58,6 +58,12 @@ func GetAttributePaths(workingDir, gitDir string) []AttributePath {
 		for scanner.Scan() {
 			line := scanner.Text()
 
+			line := strings.TrimSpace(line)
```

Comment:
The [tests](https://ci.appveyor.com/project/lfs-build/git-lfs/build/1.0.2361) show a compilation error here:

```
git\attribs.go:61: no new variables on left side of :=
git\attribs.go:63: not enough arguments in call to strings.HasPrefix
	have (string)
	want (string, string)
```

The `line` var is being initialized on line 59, and Go won't let you re-initialize it with `:=` on line 61. One of these should fix it:

```go
line := scanner.Text()
line = strings.TrimSpace(line)

// 1 liner is fine too
line := strings.TrimSpace(scanner.Text())
```